opentech/apply/review/models.py

```diff
@@ -72,3 +74,25 @@ def __str__(self):
 
     def __repr__(self):
         return f'<{self.__class__.__name__}: {str(self.review)}>'
+
+
+@receiver(post_save, sender=Review)
+def update_submission_reviewers_list(sender, **kwargs):
+    review = kwargs.get('instance')
+
+    if not review.submission.reviewers.filter(id=review.author.id).exists():
+        review.submission.reviewers.add(review.author)
+        review.submission.save()
+
+        Activity.actions.create(
+            user=review.author,
+            submission=review.submission,
+            message=f'Added themselves to the Reviewers list.'
+        )
```

Comment:
I don't think we want to track this. It feel like it should be an implicit part of submitting a review if they aren't in the list.

`Dan added himself to reviewers`
`Dan submitted a review`